cmake/CPM.cmake

```diff
@@ -648,19 +648,21 @@ function(CPMAddPackage)
     return()
   endif()
 
-  if(CPM_USE_LOCAL_PACKAGES OR CPM_LOCAL_PACKAGES_ONLY)
-    cpm_find_package(${CPM_ARGS_NAME} "${CPM_ARGS_VERSION}" ${CPM_ARGS_FIND_PACKAGE_ARGUMENTS})
+  if(NOT DEFINED CPM_ARGS_SOURCE_DIR)
```

Comment:
```suggestion
  if(NOT CPM_ARGS_FORCE)
```

Reply:
Neither `FORCE` nor `SOURCE_DIR` are documented?

And the `SOURCE_DIR` is use at `ModernCppStarter`!

Reply:
> Neither `FORCE` nor `SOURCE_DIR` are documented yet?

`SOURCE_DIR` is an origin parameter of [ExternalProject](https://cmake.org/cmake/help/latest/module/ExternalProject.html#id1), so documentation can be found there. `FORCE` we should definitely document though.